draft-kazuho-httpbis-priority.md

```diff
@@ -369,6 +364,12 @@ PRIORITY_UPDATE frame can be sent. In HTTP/2 the frame is on stream zero and
 in HTTP/3 it is sent on the
 control stream ({{!I-D.ietf-quic-http}}, Section 6.2.1).
 
+This document specifies a new PRIORITY_UPDATE frame type for HTTP/2
+({{!RFC7540}}) and HTTP/3 ({{!I-D.ietf-quic-http}}) which enables
+reprioritization. It carries updated priority parameters and references the
+target of the reprioritization based on a version-specific identifier; in
+HTTP/2 this is the Stream ID, in HTTP/3 this is either the Stream ID or Push ID.
+
```

Comment:
I'm not sure if we want to shuffle the paragraphs here, because we are now talking about how to transmit the PRIORITY_UPDATE frame before explaining what the frame is.

Reply:
Addressed by the later commits.